### PR TITLE
[DI] Improve test setup by allowing breakpoint URL to be dynamic

### PR DESCRIPTION
**Pull request by watson**
This is a required change in order to make the testing infrastructure more flexible, which is needed by upcoming PR's. I separated it into its own PR to make the review easier.

### CODE REVIEW

**Comment by watson** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-js/4996?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4996** <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-js/4996?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-js/4996?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#4992** <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-js/4992?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4991** <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-js/4991?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/dd-trace/test/debugger/devtools_client/utils.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict'
```
Comment (uurien) [inline packages/dd-trace/test/debugger/devtools_client/utils.js:1]: Why are you moving this method out of integration-test directory? Is it expected to be used in non-integration tests?

File: packages/dd-trace/test/debugger/devtools_client/utils.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const { randomUUID } = require('node:crypto')
+
+module.exports = {
```
Comment (uurien) [inline packages/dd-trace/test/debugger/devtools_client/utils.js:5]: Usually, we write `module.exports` at the end of the file.

File: packages/dd-trace/test/debugger/devtools_client/utils.js (hunk 1 above)
Comment (watson) [reply packages/dd-trace/test/debugger/devtools_client/utils.js:1]: I need to access it in [an upcoming PR](https://github.com/DataDog/dd-trace-js/pull/5004) that adds benchmarks. I thought the test directory was a better place for it than the integration tests directory

File: packages/dd-trace/test/debugger/devtools_client/utils.js (hunk 2 above)
Comment (uurien) [reply packages/dd-trace/test/debugger/devtools_client/utils.js:5]: enforced pattern I don't know tbh. It is what I see in other files, there is not a rule in the linter, so I think that it is not enforced pattern. It just catches my attention.

**Review by uurien — APPROVED** [review]

**Pull request merged**